foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/util/BrowserUtility.java

```diff
@@ -37,7 +75,7 @@
 	 * @param log
 	 *            the log provider to log against if an error occurred.
 	 */
-	public void checkedCreateInternalBrowser(String url, String browserId, String pluginId, ILog log) {
+	public static void checkedCreateInternalBrowser(String url, String browserId, String pluginId, ILog log) {
```

Comment:
but..why? early on in foundation we agreed to avoid/reduce need for singletons in here.

Seems this brings no value, just limit its use IMO.